pkgs/tools/typesetting/asciidoctor/default.nix

```
@@ -1,59 +1,29 @@
-{ stdenv, lib, bundlerApp, ruby, bundix, mkShell
-  # Dependencies of the 'mathematical' package
-, cmake, bison, flex, glib, pkgconfig, cairo
-, pango, gdk_pixbuf, libxml2, python3, patchelf
-}:
+{ lib, bundlerApp, mkShell, bundix, defaultGemConfig }:
```

```suggestion
{ lib, bundlerApp, mkShell, bundix }:
```
defaultGemConfig doesn't seem to be used